### ISSUE TITLE
Cannot be used on angle bracket components below Ember 3.8

### ISSUE THREAD

**Issue by amk221** [buschtoens/ember-on-modifier#119]
```
<MyComponent {{on "click" this.doSomething}} />
```

Fails with `Element modifiers are not allowed in components`

Demo repo: https://github.com/amk221/-ember-on-modifier/commit/5d9ae9f4cb162e8b9166904c387ccf6d5242554b

**Comment by amk221** [comment]
I realised this addon doesn’t have -polyfill in the name, is that why it doesn’t work on the lower versions?

**Comment by amk221** [comment]
I see now that this is due to missing behaviour to do with splattributes, not the `on` modifier.

**Comment by buschtoens** [comment]
Hey @amk221! Thanks for making this explicit.

If you want, you can submit a PR to add this to [Compatibility](https://github.com/buschtoens/ember-on-modifier#compatibility). If you got no time for it, that's also fine. I'll get to it myself then.

**Comment by rwjblue** [comment]
Yeah, I really wish we could polyfill that behavior but it is quite hard.
